test/e2e/apimachinery/flowcontrol.go

```diff
@@ -271,13 +206,123 @@ var _ = SIGDescribe("API priority and fairness", func() {
 				framework.Failf("client %q: got %.1f%% completed requests, want at least 95%%", client.username, 100*fractionCompleted)
 			}
 		}
+		wg.Wait()
```

Comment:
why isn't the wait on new line 197 enough?

Reply:
ah not sure how that crept in, fixing